src/jira/client/axios.test.ts

```diff
@@ -72,6 +72,24 @@ describe("Jira axios instance", () => {
 				const serialisedError = JSON.stringify(error);
 				expect(serialisedError).not.toContain(requestPayload);
 			});
+
 		});
 	});
+
+	it("should check if Jira is deactivated on failed request", async () => {
+		const requestPayload = "TestRequestPayload";
+		jiraNock.post("/foo/bar", requestPayload).reply(503);
+		jiraNock.get("/status").reply(503);
+
+		let error;
+		try {
+			await getAxiosInstance(jiraHost, "secret").post("/foo/bar", requestPayload);
+		} catch (e) {
+			error = e;
+		}
+
+		expect(error?.status).toEqual(404);
+		expect(error?.message).toEqual("Error executing Axios Request HTTP 404 - Bad REST path, or Jira instance not found or temporarily suspended.");
+	});
```

Comment:
You should also check that if an error happens with the original request, but the instance returns 200, that it still throws the original error :)